### PR TITLE
Ignore Draft PRs in .hackstats

### LINKED ISSUE #304: Ignore draft PRs for .hackstats
@SebastiaanZ brought up that not all PRs counted by `.hackstats` are valid in the DigitalOcean event due to draft PRs being ignored.

## Implementation details

- [x] Ignore draft Pull Requests returned by the GitHub API.

### PR DESCRIPTION
**Pull request by kosayoda**
**Closes #304**

Draft Pull Requests still count as valid in the current `.hackstats` command but do not for the official DigitalOcean event.

This is mitigated by checking for the `draft` field and ignoring PRs where it is `True`.

### CODE REVIEW

File: bot/seasons/halloween/hacktoberstats.py [hunk 1]
```diff
@@ -264,6 +271,7 @@ def build_embed(self, github_username: str, prs: List[dict]) -> discord.Embed:
                         ),
                     }
                     outlist.append(itemdict)
+                logging.info(f"Found {len(outlist)} valid Hacktoberfest PRs for GitHub user: '{github_username}'")
```
Comment (Numerlor) [inline bot/seasons/halloween/hacktoberstats.py]: The logging from here will be the same as the one for total issues most of the time and imo the command has a lot of logging now.
Have you considered merging those two to a single log entry?
`f"Found {len(jsonresp['items'])} total and {len(outlist)} valid Hacktoberfest PRs for GitHub user: '{github_username}'"`

Comment (kosayoda) [reply bot/seasons/halloween/hacktoberstats.py]: That's a good point, I'll batch this change with any review changes.

**Comment by kosayoda** [comment]
> But instead of looping over the PRs, why you don’t just change the search query to include `-is:draft is:pr`?

I can't find the relevant API documentation, could you link it to me?

**Comment by Akarys42** [comment]
Well, the `-` invert the search, but it isn’t documented anywhere as far as
I know, but... it work.

**Comment by Akarys42** [comment]
Well, ELA found the documentation:

Inversion: https://help.github.com/en/articles/understanding-the-search-syntax#exclude-certain-results
Draft: https://help.github.com/en/articles/searching-issues-and-pull-requests#search-for-draft-pull-requests

**Pull request merged**